Mathlib/Topology/UniformSpace/Uniformizable.lean

```diff
@@ -0,0 +1,133 @@
+/-
+Copyright (c) 2025 Aaron Liu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Liu
+-/
+module
+
+public import Mathlib.Topology.Separation.CompletelyRegular
+
+import Mathlib.Topology.UniformSpace.OfCompactT2
+import Mathlib.Topology.UrysohnsLemma
+
+/-!
+# Uniformizable Spaces
+
+A topological space is uniformizable (there exists a uniformity that
+generates the same topology) iff it is completely regular.
+
+TODO: Explain proofs
```

Comment:
Should we wait for this?

Reply:
Yes, some explanation is preferred here.